CONTRIBUTING.md

```diff
@@ -60,6 +62,16 @@ The `rustup::currentprocess` module abstracts the global state that is
 `std::env::current_dir` and `std::process::exit` permitting threaded tests of
 the CLI logic; use `process()` rather than those APIs directly.
 
+### Clippy lints
+
+We don't enforce lint-free-ness in the gate, because it is a moving target, but
+we do ask that people run `cargo +nightly clippy --all-targets -- -D warnings`
+before submitting their code, to help keep things generally tidy.
+
+Note that ideally lint checking is done on at least Linux and Windows, as
+OS-conditional code is a common source of unused imports and other small lints,
+which can build up over time.
```

Comment:
Is this directed at a contributor? I am reading this as "you should check linting on both Linux and Windows" but I am not sure how I would actually do that without a Windows machine. Do you mean examining the CI output?